src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartMapCustomGroupsConfiguration.tsx

```diff
@@ -75,7 +75,11 @@ export default function ChartMapCustomGroupsConfiguration({
                   return !(
                     groups.length &&
                     groups.some(group => {
-                      return value >= group.min && value <= group.max;
+                      return (
+                        (value >= group.min && value <= group.max) ||
+                        // eslint-disable-next-line react/no-this-in-sfc
+                        (value <= group.max && group.min <= this.parent.max)
```

Comment:
This condition is a little gnarly to me. Is there a potential issue with this resolving to true if the max field hasn't been filled in yet?

Can we also add a comment explaining what's going on here?